contributing/start.rst

```diff
@@ -56,5 +99,135 @@ Thanks! ❤ ❤ ❤
 
   Zammad Team
 
+.. _translation:
+
+Translation
+============
+
+If you want to contribute to the translations and improve the multi-language
+support of Zammad and/or the documentation, you are welcome to contribute
+as well! The translation of Zammad itself and the documentation is done by using
+Weblate, which is a service for the collaborative translation of projects.
+
+You just have to head over to Zammad in Weblate_. You can either create an
```

Comment:
`Zammad in Weblate` sounds a bit wrong. We host our own instance of Weblate at `translations.zammad.org`, so perhaps it would be better like: `Zammad's Weblate instance` or `app` or whatever.